internal/server/iigo.go

```diff
@@ -21,7 +21,7 @@ func (s *SOMASServer) runIIGO() error {
 }
 
 func (s *SOMASServer) updateIIGOTurnHistory(clientID shared.ClientID, pairs []rules.VariableValuePair) {
-	s.gameState.IIGOHistory = append(s.gameState.IIGOHistory,
+	s.gameState.IIGOHistory[s.gameState.Turn] = append(s.gameState.IIGOHistory[s.gameState.Turn],
```

Comment:
does this not crash when `s.gameState.IIGOHistory[s.gameState.Turn]` is unset?

Reply:
I thought with nil lists it creates a new one?

Reply:
ah, ok, didn't know thgis

Reply:
Oh wait, as in that entry itself is unset, hmmmmmm, better put a guard in

Reply:
> ah, ok, didn't know thgis

Ah wait soz, I misunderstood your question. I think you're right, I've put a guard in now